PWGCF/TwoParticleCorrelations/Tasks/diHadronCor.cxx

```diff
@@ -587,6 +629,57 @@ struct DiHadronCor {
   }
 
   PROCESS_SWITCH(DiHadronCor, processMixed, "Process mixed events", true);
+
+  int getSpecies(int pdgCode)
+  {
+    switch (pdgCode) {
+      case 211: // pion
+      case -211:
+        return 0;
+      case 321: // Kaon
+      case -321:
+        return 1;
+      case 2212: // proton
+      case -2212:
```

Comment:
Please don't use magic numbers
Use named constants from `TPDGCode.h`
I would also suggest defining named constants for the species internal codes